### PR TITLE
Add katello-certs-tools from katello-packaging

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: katello/katello-certs-tools/katello-certs-tools.spec [hunk 1]
```diff
@@ -0,0 +1,123 @@
+Name:      katello-certs-tools
+Summary:   Katello SSL Key/Cert Tool
+Group:     Applications/Internet
+License:   GPLv2
+Version:   2.4.0
+Release:   1%{?dist}
+URL:       https://fedorahosted.org/katello
```
Comment (ekohl) [inline katello/katello-certs-tools/katello-certs-tools.spec]: Should this be katello.org?

**Comment by ehelms** [comment]
Updated

File: katello/katello-certs-tools/katello-certs-tools.spec [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+%{python_sitelib}/*
+%dir %{_sysconfdir}/pki/%{name}
+%dir %{_sysconfdir}/pki/%{name}/certs
+%dir %{_sysconfdir}/pki/%{name}/private
+%attr(755,root,root) %{_bindir}/katello-sudo-ssl-tool
+%attr(755,root,root) %{_bindir}/katello-ssl-tool
+%attr(755,root,root) %{_bindir}/katello-certs-gen-rpm
+%attr(755,root,root) %{_bindir}/katello-certs-sign
+%doc %{_mandir}/man1/katello-*.1*
+%doc LICENSE
```
Comment (ekohl) [inline katello/katello-certs-tools/katello-certs-tools.spec]: %license?

**Comment by ehelms** [comment]
Updated

**Pull request merged**